src/manage/create-project/index.md

```diff
@@ -78,54 +78,59 @@ The project will be saved in the selected workspace.
+   If you are uploading files manually through the <DashboardShortLink />, do not forget to synchronise the changes by clicking **Update changes** in the **Data Sync**.
+   ![Upload files to Mergin Maps project](./dashboard-upload-new-project.jpg "Upload files to Mergin Maps project")
 
-### Clone an existing project through merginmaps.com
-You can also make a copy of projects you have access to or of a [public project](../permissions/#public-and-private-projects).
+### Clone an existing project through Mergin Maps dashboard
+Projects you have access to and [public projects](../permissions/#public-and-private-projects) can be cloned. By cloning a project, you will create your own copy of this project. This way, you can use public projects as templates for your own work or easily create backups.
 
 #### Clone a project from your workspace
 1. In the **Projects** tab, find the project you want to clone, open it and click on **Clone**.
 
    If needed, [switch to another workspace](../workspaces/#switch-workspaces-in-qgis) to locate the project.
    
-   ![Cloning project from your workspace](./web-clone-your-prj.jpg "Cloning project from your workspace")
+   ![Mergin Maps dashboard Cloning project from your workspace](./dashboard-clone-project.jpg "Mergin Maps dashboard - Cloning project from your workspace")
+
+2. Enter the name of the cloned project. If needed, change the target workspace.
 
-2. Enter the name of the cloned project and choose the workspace in which the project should be saved. Confirm by clicking **Clone**.
-   ![Naming cloned project](./web-clone-your-prj-2.jpg "Naming cloned project")
+   The cloned project will be saved to the workspace you have specified.
+  
+   Confirm by clicking **Clone project**.
+   ![Naming cloned project](./dashboard-clone-project-name.jpg "Naming cloned project")
 
-The new project will be saved to the workspace you have specified.
 
 #### Clone a public project
 You can also make a copy of a public project. 
 
-1. In the **Projects** tab, click on the **Find Public Projects** button
-   ![Find public Mergin Maps projects](./web-find-public-project.jpg "Find public Mergin Maps projects")
+1. In the **Projects** tab, click on the **Browse community projects** button
+   ![Mergin Maps dashboard browse community projects](./dashboard-browse-public-projects.jpg "Mergin Maps dashboard browse community projects")
    
    If a project was shared with you via link, open the project link in your web browser.
    
-2. Using the search bar, find the project you would like to clone and click on it.
-   ![Find public Mergin Maps projects](./web-find-project.jpg "Find public Mergin Maps projects")
+2. You will see a list of public projects. Use the search bar to find a project you would like to clone and click on it.
+   ![Find public Mergin Maps projects](./dashboard-find-public-project.jpg "Find public Mergin Maps projects")
 
 3. The project's page opens. Click on the **Clone** button to clone the project.
```

Comment:
Edit text:
3. When the project's page opens, click on the **Clone** button to clone the project.